src/components/Calculator.js

```diff
@@ -1,68 +1,62 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: null,
-      next: null,
-      operation: null,
-    };
-    this.valueOperation = this.valueOperation.bind(this);
-    this.handleClick = this.handleClick.bind(this);
-  }
+const Calculator = () => {
+  const objCal = {
+    total: null,
```

Comment:
- [ ] The app generates the below error when the browser is loaded. Kindly check and fix it.
![Screenshot from 2022-02-12 00-33-10](https://user-images.githubusercontent.com/20245164/153673464-98b85d3c-e465-4648-8cd7-98bceb8db896.png)
- [ ] [**Optional**] It would be better to also refactor the `app` component to a functional component.